rest-api/src/main/resources/paths/admin_exporter3/v1_admin_exporter3_apps_self_healthdata.yml

```diff
@@ -0,0 +1,19 @@
+get:
+    operationId: getRecordsEx3ForCurrentApp
+    summary: Retrieves all records from the Exporter 3 implemention in the current app for the given time range.
+    tags:
+        - _For Superadmins
```

Comment:
Internal? I know the super admins client is practically the same since we're the only two super admins...